### ISSUE TITLE
JSON.parse: unexpected character at line 1 column 1 of the JSON data

### ISSUE THREAD

**Issue by jd440** [johngodley/redirection#659]
### Summary

Just after upgrade I got the following error in back-office:
`JSON.parse: unexpected character at line 1 column 1 of the JSON data`

Data still in mysql

### Actual Behavior
I got the error on Backoffice.
On Front-office, redirection still working but as I got WP super cache, I guess redirection have been catched by cache module. As doesn't want to mess, my website I didn't want clear cache.

### Steps to reproduce
??? just upgrade

### Environment
```
Plugin: 3.0.1
WordPress: 4.9.2
PHP: 7.0.5
Browser: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:57.0) Gecko/20100101 Firefox/57.0
BA
Error: JSON.parse: unexpected character at line 1 column 1 of the JSON data
Action: redirection/v1/plugin GET
Code: 404 Not Found
```

### Fix
For the moment I Just re-upload previous version of module

**Comment by johngodley** [comment]
Can you try the 3.1 beta?

https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by johngodley** [comment]
Yes, you've totally disabled the REST API. You'll need to re-enable at the very least the cookie stuff. The rest should be fine with 3.1 I think

**Comment by jd440** [comment]
yes, I doesn't use any external service so I don't see any use of REST, exept for scapper, spammer…

by re-enable cookie stuff do you mean
```
//remove_action('auth_cookie_malformed', 'rest_cookie_collect_status');
//remove_action('auth_cookie_expired', 'rest_cookie_collect_status');
//remove_action('auth_cookie_bad_username', 'rest_cookie_collect_status');
//remove_action('auth_cookie_bad_hash', 'rest_cookie_collect_status');
//remove_action('auth_cookie_valid', 'rest_cookie_collect_status');
```

**Comment by johngodley** [comment]
Yes I think that should be enough. Scammers or scrapers shouldn't find anything in the REST API that they can't already get from your site

**Comment by johngodley** [comment]
I've now released version 3.1. I will close this ticket for now, but please reopen if the problem persists
